api/spec/v1.yaml

```diff
@@ -2113,23 +2113,28 @@ components:
         verification:
           $ref: '#/components/schemas/RuntimeEvmContractVerification'
           description: |
-            Additional information obtained from contract verification. Only available for smart 
+            Additional information obtained from contract verification. Only available for smart
             contracts that have been verified successfully by Sourcify.
-    
+
     RuntimeEvmContractVerification:
       type: object
-      # properties:
-      #   abi:
-      #     type: object
-      #     description: |
-      #       The smart contract's [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI), in JSON format as defined by Solidity. Available if 
-      #       and only if the contract has been "verified" through [Sourcify](https://sourcify.dev), i.e. Sourcify has checked 
-      #       and confirmed that the source code and ABI correspond to the contract's creation bytecode.
-      #   compiler_version:
-      #     type: string
-      #     description: |
-      #       The version of the compiler used to verify the smart contract.
-      #     example: 0.8.18+commit.87f61d96
+      properties:
+        abi:
+          type: array
+          items:
+            type: object
+          description: |
+            The smart contract's [ABI](https://docs.soliditylang.org/en/develop/abi-spec.html), in JSON format as defined by Solidity.
+        metadata_json:
```

Comment:
Don't spend too long on this bikeshedding comment, but should this be
- `metadata_json` (nice in that it reflects the originating file name; bad in that it doesn't mean much on its own, and in that it includes the type (`_json`) in the name, which fields typically don't), or
- `compilation_metadata` or similar?

I'm fine with either, just pointing out the oddity of naming in case you have stronger opinions. The commented-out names were just placeholders, we're not beholden to them.

Reply:
Maybe `contract_metadata`? That's also what Sourcify calls it.

Reply:
Updated to `compilation_metadata`